mlir/test/fusion/e2e/resnet50/mixr-resnet-fusion-case-2.nonclone.mlir

```diff
@@ -0,0 +1,30 @@
+  // RUN: rocmlir-opt -migraphx-to-tosa %s | rocmlir-driver -host-pipeline highlevel | rocmlir-gen -ph -print-results -rand none - | rocmlir-driver -arch %arch -c | mlir-cpu-runner -O2 --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext --entry-point-result=void | FileCheck %s
```

Comment:
This is my bespoke verification method pre-clone era and it works.
This is a test case migraphx succeeds in compilation (well; verification has issues but thats a different issue).